Week1Day5

Raja-Tahseen opened the pull request:
**What this does**
With this PR, I've implemented some javascript string functionalities.

**Implementation**
Created an Week1Day5.js and Week1Day5.md

**Testing**
![image](https://github.com/user-attachments/assets/6757bac3-7788-4edd-afd0-9b7b2ad26b34)
![image](https://github.com/user-attachments/assets/360a9988-fcf5-43a0-9a1b-1c36054e27b4)

Monfernape commented on Week1Day5.md:
```diff
@@ -0,0 +1,107 @@
+## 1. How to check the length of a string?
+
+To check the length of a string in JavaScript, you can use the `length` property.
```
What we actually want here is real world use case

Raja-Tahseen replied:
**Thank you** for the feedback! I’ve updated the examples to include **12 real-world use cases** (like masking credit cards, formatting phone numbers, filtering emails, etc.) to better align with practical scenarios. Let me know if you’d like me to tweak any of these further—I’m happy to refine! 🙌

Monfernape commented on Week1Day5.md:
```diff
@@ -0,0 +1,107 @@
+
+### Key Points
+
+1. If startIndex is greater than endIndex, slice will return an empty string.
```
Nice 🔥

Monfernape reviewed (approved):
Looks good, minor comments to take care of 👌

Raja-Tahseen commented:
**What this does**
Next, I've implemented **Number Methods** section coding challenges and also related .md file.

**Implementation**
Updated Week1Day5.js and Created  Week1Day5_Number Methods.md

**Testing**
![image](https://github.com/user-attachments/assets/55a16648-be25-4291-aa45-b9f1df361dbb)
![image](https://github.com/user-attachments/assets/2cc1d2aa-c7a5-4eb3-9431-cfc6ee5b3e3f)
![image](https://github.com/user-attachments/assets/f7ad4bc6-e34a-4be2-9743-fefe820982c1)
![image](https://github.com/user-attachments/assets/102fa469-11ef-4b6d-b5f2-c817064cda77)

Raja-Tahseen commented:
**What this does**
Next, I've implemented Math Methods section coding challenges and also related .md file.

**Implementation**
Updated Week1Day5.js and Created Week1Day5_Math_FUnctions.md

**Testing**
![image](https://github.com/user-attachments/assets/b899f308-7e97-4e10-b839-e83306897809)
![image](https://github.com/user-attachments/assets/a786300f-4142-4962-af6a-3e302974fed7)
![image](https://github.com/user-attachments/assets/e0bac0c5-0b60-4033-99cf-9c538e348e5e)

Monfernape commented on Week1Day5.js:
```diff
@@ -0,0 +1,100 @@
+//1. Make a head/tails program.
+function flipCoin() {
+  // Returns "Heads" or "Tails" randomly
+  return Math.random() < 0.5 ? "Heads" : "Tails";
```
```js
const randomNumber = Math.random();
if(randomNumber > 0.5) {
 return 'Heads'
} else {
 return 'Tails'
}
```
